questdb-rs/src/ingress/mod.md

```diff
@@ -0,0 +1,232 @@
+# Fast Ingestion of Data into QuestDB
+
+The `ingress` module implements QuestDB's variant of the
+[InfluxDB Line Protocol](https://questdb.io/docs/reference/api/ilp/overview/)
+(ILP) over HTTP and TCP. We recommend using HTTP because it has a number of
+features that TCP lacks, such as receiving error messages from the server and
+fine-grained transaction control.
+
+To get started:
```

Comment:
this is great! 3 lines to give you the basic usage pattern! I might steal this for Java. I have something similar, but more detailed. I think yours is better for a quick overview of the expected usage pattern.